suite-native/ethereum-tokens/src/components/CoinIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { networks, NetworkSymbol } from '@suite-common/wallet-config';
+import { CryptoIcon, CryptoIconSize, EthereumTokenIcon } from '@trezor/icons';
+
+import { EthereumTokenSymbol } from '../types';
+import { getEthereumTokenIconName } from '../utils';
+
+export const CoinIcon = ({
```

Comment:
We will have to unify Icon's somehow this is starting to be hell, we have like 5 nearly similar components for Crypto icons and now in multiple packages. I would say we should merge this functionality into `CryptoIcon`  component.

Reply:
Reference to the new issue created in fba47c7bc.